content/pt-br/docs/reference/access-authn-authz/authorization.md

```diff
@@ -0,0 +1,250 @@
+
+* `SubjectAccessReview` - * `SubjectAccessReview` - Revisão de acesso para qualquer usuário, não apenas o atual. Útil para delegar decisões de autorização para o servidor de API. Por exemplo, o kubelet e extensões de servidores de API utilizam disso para determinar o acesso do usuário às suas próprias APIs.
+
+* `LocalSubjectAccessReview` - Similar a `SubjectAccessReview`, mas restrito a um namespace específico.
+
+* `SelfSubjectRulesReview` - Uma revisão que retorna o conjunto de ações que um usuário pode executar em um namespace. Útil para usuários resumirem rapidamente seu próprio acesso ou para Interfaces de Usuário ocultarem/mostrar ações.
+
+Essas APIs podem ser consultadas criando recursos normais do Kubernetes, onde a resposta `status`
+campo do objeto retornado é o resultado da consulta.
+
+```bash
+kubectl create -f - -o yaml << EOF
+apiVersion: authorization.k8s.io/v1
+kind: SelfSubjectAccessReview
+spec:
+  resourceAttributes:
+    group: apps
+    resource: deployments
+    verb: create
+    namespace: dev
+EOF
+```
+
+A `SelfSubjectAccessReview` gerada seria:
+```yaml
+apiVersion: authorization.k8s.io/v1
+kind: SelfSubjectAccessReview
+metadata:
+  creationTimestamp: null
+spec:
+  resourceAttributes:
+    group: apps
+    resource: deployments
+    namespace: dev
+    verb: create
+status:
+  allowed: true
+  denied: false
+```
+
+## Usando flags para seu módulo de autorização
+
+Você deve incluir uma flag em sua política para indicar qual módulo de autorização
+suas políticas incluem:
+
+As seguintes flags podem ser utilizadas:
+
+  * `--authorization-mode=ABAC` O modo de controle de acesso baseado em atributos [Attribute-Based Access Control (ABAC)] permite configurar políticas usando arquivos locais.
+  * `--authorization-mode=RBAC` O modo de controle de acesso baseado em função [Role-based access control (RBAC)] permite que você crie e armazene políticas usando a API Kubernetes.
+  * `--authorization-mode=Webhook` WebHook é um modo de retorno de chamada HTTP que permite gerenciar a autorização usando endpoint REST.
+  * `--authorization-mode=Node` A autorização de nó é um modo de autorização de propósito especial que autoriza especificamente requisições de API feitas por kubelets.
+  * `--authorization-mode=AlwaysDeny` Esta flag bloqueia todas as requisições. Utilize esta flag somente para testes.
+  * `--authorization-mode=AlwaysAllow` Esta flag permite todas as requisições. Utilize esta flag somente se nao existam requisitos de autorização para as requisições de API.
+
+Você pode escolher mais de um modulo de autorização. Módulos são verificados
+em ordem, então, um modulo anterior tem maior prioridade para permitir ou negar uma requisição.
+
+## Escalonamento de privilégios através da criação ou edição da cargas de trabalho {#privilege-escalation-via-pod-creation}
+
+Usuários que podem criar ou editar pods em um namespace diretamente ou através de um [controlador](/docs/concepts/architecture/controller/)
```

Comment:
It can be replaced with [pt-br](https://kubernetes.io/pt-br/docs/concepts/architecture/controller/) version.
```suggestion
Usuários que podem criar ou editar pods em um namespace diretamente ou através de um [controlador](/pt-br/docs/concepts/architecture/controller/)
```